documentation/course/E1.md

```diff
@@ -210,6 +216,7 @@ classDiagram
     class Accident {
         long id [1] #lcub;id, readonly#rcub;
         Instant time [1] #lcub;readonly#rcub;
+        Boolean resolved [1] = false
```

Comment:
**nit:** `Boolean` suggests that `resolved` could be `null` and I don't see any business reason for that to be the case.
```suggestion
        boolean resolved [1] = false
```

Reply:
ahh yes, Java